internal/config/view.go

```diff
@@ -34,6 +34,7 @@ type View interface {
 	GetBool(string) bool
 	GetDuration(string) time.Duration
 	GetStringMap(string) map[string]interface{}
+	AllSettings() map[string]interface{}
```

Comment:
AllSettings is only used against concrete `*viper.Viper` so this isn't necessary. If you need it because of tests I'd prefer a type assertion case to viper.Viper there since you already know you're testing against that class to begin with.

Reply:
Makes sense.

Reply:
Done, thanks!